packages/site/pages/contribute/index.js

```diff
@@ -146,6 +173,12 @@ The Design System is used in many places.  We have a hard time keeping our eyes
 
 ### Offer design direction
 Maybe you have a visual, interactive, or code design insight.  We’d love to chat about your ideas and see how they might fit and be used to improve the Design System.   Join [#design-system-help](slack://channel?team=pluralsight&id=design-system-help) and share.
+
+### Propose an enhancement or new pattern
+Does it belong in the Design System? Check out the new [new pattern journey](/static/img/diagrams/new-pattern-journey.png) workflow to scope whether a new pattern should be added to the Design System.
+
+![new pattern journey](/static/img/diagrams/new-pattern-journey.png)
```

Comment:
That's a giant image, kb-wise.  Anything we could do to make it smaller?

Reply:
probably - can't say i tried. i will optimized :)